examples/apps/cra-demo/README.md

```diff
@@ -98,86 +104,87 @@ const getContainerId = () => {
 };
 ```
 
-### 3.b Initialize Fluid with the appropriate service
+### 3.b Initialize the service client
 
-The Fluid import can be initialized with a number of different services. The `TinyliciousService` defines the schema for connecting to a locally deployed [tinylicious](https://www.npmjs.com/package/tinylicious) Fluid service. You'll use different provided service packages depending on the service you are connecting to.
+`TinyliciousClient` must be initialized before being used. This ensures that there is a single instance of the client running in your application. Depending on your deploy target, you might initialize and use different client packages.
```

Comment:
This isn't _quite_ accurate; tinyliciousclient needs to be initialized because it can take the optional TinyliciousConnectionConfig object, and the TinyliciousClient wrapper manages a singleton TinyliciousClientInstance object.  TinyliciousClient could be changed to not require the init call when not providing the connection config though

Reply:
okay, so it's more about initializing a connection config? That's probably easier to explain :D